app/controllers/api/v1/venues_controller.rb

```diff
@@ -0,0 +1,52 @@
+# :reek:InstanceVariableAssumption :reek:NilCheck
+class Api::V1::VenuesController < ApplicationController
+  before_action :authenticate_user!
+  before_action :organizer?, only: %i[create update destroy]
+  before_action :set_venue, only: %i[show update destroy]
+
+  def index
+    @venues = Venue.all
+    render json: @venues
+  end
+
+  def show
+    render json: @venue
+  end
+
+  def create
+    @venue = Venue.new(venue_params)
+    if @venue.save
+      render json: @venue, status: :created
+    else
+      render json: @venue.errors.full_messages, status: :unprocessable_entity
+    end
+  end
+
+  def update
+    if @venue.update_attributes(venue_params)
+      render json: @venue
+    else
+      render json: @venue.errors.full_messages, status: :unprocessable_entity
+    end
+  end
+
+  def destroy
+    @venue.destroy
+    head :no_content
+  end
+
+  private
+
+  def set_venue
+    @venue = Venue.find_by(id: params[:id])
+    head :not_found unless @venue
+  end
+
+  def organizer?
+    return render json: ['Only organizer can do this'], status: :unprocessable_entity unless current_user.organizer?
```

Comment:
we agreed to put errors under errors key in response. please update your code like 
return render json: { errors: ... }

Reply:
done, now app returning only http status 'Locked', and if this status caught by Rect it render error message for this status in frontend.